packages/teams-js/src/private/index.ts

```diff
@@ -18,7 +18,7 @@ export {
   registerUserSettingsChangeHandler,
   openFilePreview,
 } from './privateAPIs';
-export { conversations } from './conversations';
+export { conversations, OpenConversationRequest, ConversationResponse } from './conversations';
```

Comment:
What is this change needed for?

Reply:
While developing, I was running into a bug where OpenConversationRequest and ConversationResponse was not able to be imported by the teams test app, so I had to export it directly to resolve the issue. I just tried removing the change and it seems to be working ok, so I've reverted the change.